.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,48 @@
+# Note: this workflow is currently being tested,
+# and so is only activated for the benchmark-test-cjs branch
+
+name: Performance benchmarks
+
+on:
+  push:
+    branches:
+      - benchmark-test-cjs
+
+jobs:
+
+  benchmarks:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install python dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -e .[tests]
+    - name: Run benchmarks
+      run: pytest --benchmark-only --benchmark-json output.json
+    - name: Store benchmark result
+      ## Run only on push on develop! Otherwise people (or other branches) might access to the github-actions branch
+      ## This is currently disabled because we are in a workflow that has the correct 'on' settings (only push,
+      ## and only 'develop'). Otherwise, enable this
+      #if: "github.event_name == 'push' && github.ref == 'refs/heads/develop'"
+      uses: rhysd/github-action-benchmark@v1
+      with:
+        name: "Benchmark on ${{ matrix.os }}"
+        tool: "pytest"
+        output-file-path: output.json
+        github-token: ${{ secrets.GITHUB_TOKEN }}
+        auto-push: true
+        # Show alert with commit comment on detecting possible performance regression
+        alert-threshold: '200%'
+        comment-on-alert: false
+        fail-on-alert: false
```

Comment:
So what will happen if the threshold is exceeded? It seems to me that at least `comment-on-alert` should be `true`?

Reply:
Well for now it is turned off, until I have had a chance to assess some initial data as to how variable the data is.
Then the intention is that when I make a follow up PR, with some benchmarking tests implemented, I will set one/both to true with an informed threshold.

Reply:
Sounds like a good plan. 👍